esx_service/vmdk_ops.py

```diff
@@ -268,11 +282,21 @@ def getVMDKBacking(vmdk_path):
         # vsan object.
         cmd = "{0} {1}".format(OBJ_TOOL_CMD, uuid)
         rc, out = RunCommand(cmd)
-        fpath = "/vmfs/devices/vsan/{0}".format(uuid)
+        fpath = getVSANDevFSPath(uuid)
         if rc == 0 and os.path.isfile(fpath):
             return fpath
     return None
 
+def cleanupVSANDevFSNode(vmdk_path):
+    devFSNode = getVSANDevFSNode(vmdk_path)
```

Comment:
we should simply pass the devfs_path instead of reconstructing it (not relevant if you take my suggestion)